### ISSUE TITLE
Training Script

### ISSUE THREAD

**Issue by TejasKumar3005** [rorizzz/YOLO-Stutter#1]
Can you please also share the training script with audio transforms used on the dataset and loss function?

**Comment by rorizzz** [comment]
Thank you for your check. We will upload the training scripts asap.

**Comment by rorizzz** [comment]
Hi Tejas, we just uploaded our training script. Please check it under `yolo-stutter` folder.

**Comment by TejasKumar3005** [comment]
Thanks
